iocore/net/UnixNet.cc

```diff
@@ -237,6 +240,10 @@ initialize_thread_for_net(EThread *thread)
 #else
   thread->ep->start(pd, thread->evpipe[0], nullptr, EVENTIO_READ);
 #endif
+
+  SCOPED_MUTEX_LOCK(lock, startAcceptMutex, this_ethread());
+  started_et_net_threads++;
```

Comment:
Use prefix increment unless you specifically need postfix.

Reply:
ok